## given quantities

Maximum radial velocities for two components of a spectroscopic binary:

$v_{1obs} = \mathbf{100 km s^{-1}}$  
$v_{2obs} = \mathbf{200 km s^{-1}}$

Orbital period, $\tau$ = $\mathbf{2 days}$

$\mathbf{Orbits}$ $\mathbf{are}$ $\mathbf{circular}$

## mass ratio of binaries

$\frac{|v_{1obs}|}{|v_{2obs}|} = \frac{r_1}{r_2} = \frac{M_2}{M_1}$  

This straightforwardly gives us  

$\frac{M_2}{M_1} = \frac{|v_{1obs}|}{|v_{2obs}|} = \frac{1}{2}$

## M $\sin^3 i$

From Kepler's law, we get . 

$(M_1 + M_2)\sin^3 i = \tau \frac{(|v_{1obs}|+|v_{2obs}|)^3}{2\pi G}$

With $M_2 = M_1/2$, we get

$\frac{3}{2}M_1\sin^3 i = \tau \frac{(|v_{1obs}|+|v_{2obs}|)^3}{2\pi G} $
$\implies M_1 \sin^3 i = \frac{2}{3} \frac{300^3}{\pi G}\textrm{days}$  

Similarly,

$M_2 \sin^3 i = \frac{1}{3}\frac{300^3}{\pi G}\textrm{days}$

Using $G = 6.67 \times 10^{-11} m^3 kg^{-1} s^{-2}$,

In [15]:
import numpy as np
G = 6.67e-11
m = 1.98e30
frac = (300**3)*(1e3 **3)*24*3600/(np.pi*G)
val1 = 2*frac/3/m
val2 = frac/3/m
print(val1,val2)

3.748395006989166 1.874197503494583


Thus, we get

$ M_1 \sin^3 i = 3.7 M_\odot$  
$ M_2 \sin^3 i = 1.8 M_\odot$  

## mean expectation value of $\sin^3 i$

Integrated over the solid angle,

\begin{align}
<\sin^3 i> =& \frac{1}{4\pi} \int_\Omega \sin^3 i \, d\Omega \\
=& \frac{1}{4\pi} \int_0^{2\pi} d\phi \int_0^\pi \sin^3i \sin \theta \, d\theta \\
=& \frac{3\pi}{16} \sim 0.59
\end{align}

Using this value of $\sin^3 i$,

$M_1 = 6.27 M_\odot$  
$M_2 = 3.05 M_\odot$